Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty dataset in the specified project.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty dataset in the specified project from a Dataset Object.  This Dataset can be supplied by object via the 
+    /// pipeline or with the -ByObject parameter, or they can be supplied by value with the flags listed below.  If no Project is 
+    /// specified, the default project will be used.  This cmdlet returns a Dataset object.  Required value parameters are to be 
+    /// passed in order after the command, and optional parameters are passed as flags.
+    ///    ByValue Parameters: 
+    /// DatasetId (required) - unique identifier for the dataset.
+    /// Name - descriptive name for the dataset.
+    /// Description - user-friendly description of the dataset.
+    /// Timeout - default duration in ms for tables in the dataset to exist. 
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $dataset | New-GcbqDataset -Project "my-project" </code>
+    ///   <para>This command will take a Dataset object from the pipeline and insert it into the Cloud project "my-project". 
+    ///   project $projId</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> New-GcbqDataset "test_data_id" -Name "Testdata" -Description "Some interesting data!" -Timeout 86400000</code>
```

Comment:
Please split this at 80 chars (ideally 60). The code snippets look very bad on the documentation site if they are long. (Use the "`" character.)